src/utils/stores.ts

```diff
@@ -71,11 +64,9 @@ export function getOrCreateMarket(baseToken: Address): Market {
         market.baseToken = Address.fromString(ADDRESS_ZERO)
         market.quoteToken = Address.fromString(ADDRESS_ZERO)
         market.pool = Address.fromString(ADDRESS_ZERO)
-        market.feeRatio = BI_ZERO
+        market.feeRatio = BD_ZERO
```

Comment:
A random question. Is there a guideline to indicate when to use `BI_ZERO` and `BD_ZERO`?